src/client/pydaos/pydaos_core.py

```diff
@@ -334,17 +303,25 @@ def __setitem__(self, key, val):
     def __delitem__(self, key):
         self.put(key, None)
 
-    def bget(self, ddict, value_size=None):
+    def pop(self, key):
+        self.put(key, None)
+
+    def bget(self, d, value_size=None):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.